packages/next/client/vitals.ts

```diff
@@ -5,15 +5,19 @@ type ReportWebVitalsCallback = (webVitals: NextWebVitalsMetric) => any
 export const webVitalsCallbacks = new Set<ReportWebVitalsCallback>()
 
 let flushed = false
-export const bufferedVitalsMetrics: NextWebVitalsMetric[] = []
+const metrics: NextWebVitalsMetric[] = []
+
+export function getBufferedVitalsMetrics() {
+  return metrics
+}
```

Comment:
Do we need to expose the metrics themselves or can we just expose the length?

Reply:
I think it's fine to keep like this for internal usage? it's not exposed in `next/vitals` but only the hook.

Reply:
Ah, didn't realize that. Yeah that's fine then.